### PR TITLE
Add null checks to StackString usages.

### PR DESCRIPTION
**Pull request by Priya91**
@sergiy-k @joshfree

### CODE REVIEW

**Comment by Priya91** [comment]
@janvorli  @sergiy-k Can you please review these changes, thanks!

File: src/pal/src/file/file.cpp [hunk 1]
```diff
@@ -999,6 +999,12 @@ CreateFileW(
     }
     
     name = namePathString.OpenStringBuffer(length);
+    if (NULL == name)
+    {
+        palError = ERROR_NOT_ENOUGH_MEMORY;
```
Comment (sergiy-k) [inline src/pal/src/file/file.cpp]: This function does SetLastError before the "done" label. Could you please move SetLastError to be after "done:"?

**Comment by sergiy-k** [comment]
LGTM

File: src/pal/src/file/file.cpp (hunk 1 above)
Comment (Priya91) [inline src/pal/src/file/file.cpp]: fixed.

**Pull request merged**